packages/client/components/SideBarStartMeetingButton.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {useLocation} from 'react-router'
+import useRouter from '~/hooks/useRouter'
+import getTeamIdFromPathname from '~/utils/getTeamIdFromPathname'
+import {BezierCurve} from '../types/constEnums'
+import FlatPrimaryButton from './FlatPrimaryButton'
+import Icon from './Icon'
+
+const Button = styled(FlatPrimaryButton)<{isOpen: boolean}>(({isOpen}) => ({
+  height: 48,
+  overflow: 'hidden',
+  paddingLeft: isOpen ? 5 : 3,
+  paddingRight: isOpen ? 5 : 3,
+  width: isOpen ? 155 : 48,
+  marginLeft: 10,
+  marginTop: 15,
+  marginBottom: 15,
+  transition: `all 300ms ${BezierCurve.DECELERATE}`,
+  justifyContent: 'flex-start'
+}))
+
+const MeetingIcon = styled(Icon)<{isOpen: boolean}>(({isOpen}) => ({
+  marginLeft: isOpen ? 2 : 8,
```

Comment:
+1 I propose to modify the design slightly to keep the + stable, i.e. replacing this with just `8` and reducing the Buttons `marginLeft`.

Reply:
Ok